api/src/db/migrations/2024.06.07T00.36.14.clean-user-group-names.ts

```diff
@@ -1,12 +1,16 @@
 import type { Migration } from "@/db/umzug"
-import { UserGroup } from "@/models"
 
-export const up: Migration = async () => {
-  await UserGroup.findEach(async (userGroup) => {
-    await userGroup.update({
-      name: userGroup.name.trim().replace(/\s+/g, " "),
-    })
-  })
+export const up: Migration = async ({ context: queryInterface }) => {
+  await queryInterface.sequelize.transaction(async (transaction) => {
+    const [rows]: any[] = await queryInterface.sequelize.query('SELECT name from user_groups', { transaction });
+    for (const row of rows) {
+      const trimmedName = row.name.trim().replace(/\s+/g, " ")
+      await queryInterface.sequelize.query('UPDATE user_groups SET name = ? WHERE id = ?', {
```

Comment:
Purely a style suggestion here, but I usually use named replacements. e.g `:name` vs `?` as its easier to read.